WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -515,7 +530,8 @@ public void onPostUploaded(PostStore.OnPostUploaded event) {
             if (site != null) {
                 if (event.post.getLocalSiteId() == site.getId()) {
                     UploadUtils.onPostUploadedSnackbarHandler(getActivity(),
-                            getActivity().findViewById(R.id.coordinator), event, site, mDispatcher);
+                            getActivity().findViewById(R.id.coordinator),
```

Comment:
We can drop `final PostModel post = event.post;` at the top of this method.

Reply:
addressed in fcfa5c2